app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php

```diff
@@ -415,7 +415,7 @@ protected function _getGenerateSessionEnd() {
      */
     protected function _getGenerateSession() {
         return Mage::getStoreConfig( 'turpentine_varnish/general/vcl_fix' )
```

Comment:
@craigcarnell I think we should use getStoreConfigFlag instead of getStoreConfig since getStoreConfigFlag returns a boolean.